server/neptune/workflows/activities/command/subcommand.go

```diff
@@ -79,6 +83,13 @@ func NewSubCommand(op Operation) *SubCommand {
 	}
 }
 
+// WithDupArgs adds args without de-duplicating them for
+// commands that allow for multiple identical arg keys to exist
+func (c *SubCommand) WithDupArgs(args ...Argument) *SubCommand {
```

Comment:
If we need to do this i think `WithArgs` should not worry about duplicates and have something like `WithUniqueArgs` for the other case.

`WithDupArgs` is weird because its more general, yet we're creating a specific function for it. In hindsight i probably should have just named WithArgs more specific.